modules/sandboxed-containers-uninstalling-kata-runtime.adoc

```diff
@@ -0,0 +1,20 @@
+//Module included in the following assemblies:
+//
+// *disabling-sandboxed-container-workloads.adoc
+
+[id="sandboxed-containers-uninstalling-kata-runtime_{context}"]
+
+= Uninstalling Kata Runtime
```

Comment:
"Kata Runtime" here, but "Kata runtime" in sandboxed-containers-triggering-installation-kata-runtime.adoc  (I think lowercase is correct?) Also "**the** Kata runtime" (with article) would be consistent with the rest.